tree-walk.c

```diff
@@ -521,19 +528,27 @@ static int find_tree_entry(struct tree_desc *t, const char *name, struct object_
 			oidcpy(result, oid);
 			return 0;
 		}
-		return get_tree_entry(oid, name + entrylen, result, mode);
+		return get_tree_entry_gently(oid, name + entrylen, result, mode, gently);
 	}
 	return -1;
 }
 
-int get_tree_entry(const struct object_id *tree_oid, const char *name, struct object_id *oid, unsigned *mode)
+int get_tree_entry_gently(const struct object_id *tree_oid, const char *name, struct object_id *oid, unsigned *mode, int gently)
```

Comment:
I guess that the best way would be to do as `Documentation/CodingGuidelines` states. In a future patch (since it is not related to `GET_OID_GENTLY`), I will fix the existing tiny mistakes regarding code styling.

I will edit this.